docs/extensions/tutorials/_posts/1970-01-01-GettingStarted.md

```diff
@@ -88,34 +91,34 @@ Initializing extension:
 Extension initialized!
 ```
 
-The `shoutem init` command bootstrapped the `hello-world` folder with extension files. Locate to the extension folder:
+The `shoutem init` command bootstrapped the `hello-world` folder with extension files. Switch over to the extension folder:
 
 ```ShellSession
 $ cd hello-world
 ```
 
-Create a screen with a shortcut (pointer to the starting screen of an extension):
+Create a screen with a shortcut (pointer to the starting screen of the extension):
 
 ```ShellSession
 $ shoutem screen add Hello --shortcut Hello
-Enter shortcut information:
+File `app/screens/Hello.js` created.
+Enter shortcut information.
 Title: Hello
-Screen `List` is created in file `app/screens/List.js`!
-Shortcut `List` is created!
-Shortcut `List` opens `List` screen.
-File `app/extension.js` was modified.
-File `extension.json` was modified.
+File `extension.json` was modified
+Shortcut `Hello` is created.
```

Comment:
I'll revert the CLI output, but I'll still change "List" into "Hello". Oversight in the original documentation, probably copy-pasted from My First Extension where you make a screen called List with a "Restaurants" shortcut.

```
$ shoutem screen add Hello --shortcut Hello
Enter shortcut information:
Title: Hello
Screen `Hello` is created in file `app/screens/Hello.js`!
Shortcut `Hello` is created!
Shortcut `Hello` opens `Hello` screen.
File `app/extension.js` was modified.
File `extension.json` was modified.
```